snapshot/restore/local_fs_output.go

```diff
@@ -299,7 +299,30 @@ func (o *FilesystemOutput) createDirectory(ctx context.Context, path string) err
 	}
 }
 
-func (o *FilesystemOutput) copyFileContent(ctx context.Context, targetPath string, f fs.File) error {
+func write(targetPath string, r fs.Reader) error {
+	f, err := os.Create(targetPath)
+	if err != nil {
+		return errors.Errorf("unable to create file %q: %v", targetPath, err)
```

Comment:
Use `errors.Wrap()` or in this case return `err` directly since filesystem errors already include path in the text.